### PR TITLE
Fix issues with support for relative paths

### PR DESCRIPTION
**Pull request by benthorner**
[^1] contained a couple of major flaws:

- The tests referenced in the PR were not committed.
- Part of the new code contained a bug with a variable.

This fixes both of these issues.

### CODE REVIEW

File: autoload/python_copy_reference.vim
Comment (ranelpadon) [inline autoload/python_copy_reference.vim:65]: Good catch, and this will be only triggered if the `paths` key is not set, that's why didn't encounter issue also in my local.

And could we rename the `paths` setting to `remove_prefixes` as discussed here?
https://github.com/ranelpadon/python-copy-reference.vim/pull/1#issuecomment-1369414402

I think would be good to have that change before we merge this PR, unless you feel it should be in a separate PR.

File: test/vimrc
Comment (ranelpadon) [inline test/vimrc:3]: Thanks for adding the test-related files!

File: autoload/python_copy_reference.vim
Comment (benthorner) [reply autoload/python_copy_reference.vim:65]: Sure, I've added another commit to change it, and one more to mention it in the README.

Comment (ranelpadon) [reply autoload/python_copy_reference.vim:65]: Great, looks much better now. Thanks a lot! :)

**Review by ranelpadon — APPROVED** [review]

**Pull request merged**
